astropy/wcs/wcsapi/tests/test_fitswcs.py

```diff
@@ -914,3 +915,18 @@ def test_sub_wcsapi_attributes():
     assert wcs_sub4.world_axis_physical_types == ['pos.galactic.lon', 'pos.galactic.lat']
     assert wcs_sub4.world_axis_units == ['deg', 'deg']
     assert wcs_sub4.world_axis_names == ['', '']
+
+
+def test_low_level_types(header_time_1d):
+    pytest.importorskip("typeguard")
+    from .utils import validate_low_level_wcs_types
+
+    # TODO: make all these fixtures
+    wcses = [WCS_TIME_CUBE, WCS_EMPTY, WCS_SIMPLE_CELESTIAL]
```

Comment:
For this PR, can you at least just parameterize over these?

Reply:
it's a pain because of the other one that needs to be put in the warnings filter. I would rather just find the time to clean up the whole file.